plugins/producer_plugin/producer_plugin.cpp

```diff
@@ -428,6 +428,7 @@ class producer_plugin_impl : public std::enable_shared_from_this<producer_plugin
          } catch ( boost::interprocess::bad_alloc& ) {
             chain_plugin::handle_db_exhaustion();
          } catch( const fc::exception& e ) {
+            elog((e.to_detail_string()));
```

Comment:
This should be `fc_dlog` and I would add a bit of description for context.

Reply:
Done.